examples/specs/normalized/stacked_bar_count_corner_radius_stroke_normalized.vl.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
+  "data": {"url": "data/seattle-weather.csv"},
+  "mark": {"type": "bar", "cornerRadius": 10, "stroke": "black"},
+  "encoding": {
+    "x": {"timeUnit": "month", "field": "date", "type": "ordinal"},
+    "y": {"aggregate": "count", "field": "*", "type": "quantitative"},
+    "color": {"field": "weather", "type": "nominal"}
+  },
+  "config": {"bar": {"strokeWidth": 2}},
+  "autosize": {"type": "pad"}
```

Comment:
Seeing this file unnecessarily generated makes me wonder if it was a good idea that we did https://github.com/vega/vega-lite/pull/5457.

(With that refactor, now our PRs will become generally more complicated as we always generate a normalized version of all specs.)

This is beyond the scope of this PR though. :)

Reply:
I don't want to always create a normalized version. I don't know why I accepted it. I think we should fix this. I filed https://github.com/vega/vega-lite/issues/5618